grapple/actions.py

```diff
@@ -115,9 +115,47 @@ def get_fields_and_properties(cls):
     return fields + properties
 
 
-def model_resolver(field_source):
+def get_field_type(field):
+    # If a tuple is returned then obj[1] wraps obj[0]
+    field_wrapper = None
+    if isinstance(field, tuple):
+        field, field_wrapper = field
+        if callable(field):
+            field = field()
+
+    field_type = field.field_type
+    if field_type is not None:
+        if field_wrapper:
+            return field, field_wrapper(field_type)
+        else:
+            if callable(field_type):
+                field_type = field_type()
+            return field, field_type
+
+
+def model_resolver(field):
     def mixin(self, instance, info, **kwargs):
-        return getattr(instance, field_source)
+        from .utils import resolve_queryset
+
+        cls_field = getattr(instance, field.field_source)
+
+        # If queryset then call .all() method
+        if hasattr(cls_field, "all"):
```

Comment:
are there cases where there'd be an `all` attribute, but this is not a queryset?

Reply:
Now use: `if issubclass(type(cls_field), models.Manager):`